devicetypes/smartthings/ezex-temp-humidity-sensor.src/ezex-temp-humidity-sensor.groovy

```diff
@@ -0,0 +1,118 @@
+/**
+ *  eZEX Temp & Humidity Sensor(AC Type)
+ *
+ *  Copyright 2019 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "eZEX Temp & Humidity Sensor", namespace: "smartthings", author: "SmartThings", runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false, mnmn:"SmartThings", vid:"generic-humidity") {
```

Comment:
Can you remove `runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false,`? We usually don't put that stuff in if it doesn't run locally